integration/cosmosgen/testdata/bank_test.ts

```diff
@@ -0,0 +1,61 @@
+import { beforeAll, describe, expect, test } from 'vitest'
```

Comment:
Can we move the tests to integration/ dir?

Reply:
In any case it is possible to have the `*_test.ts` files anywhere we want. We can also change the file name suffix, which at the moment is "_test" to match with Go test file name suffix.